lib/inspec/utils/telemetry/http.rb

```diff
@@ -1,11 +1,16 @@
 # frozen_string_literal: true
 require_relative "base"
 require "faraday" unless defined?(Faraday)
+require "inspec/utils/licensing_config"
 module Inspec
   class Telemetry
     class HTTP < Base
       TELEMETRY_JOBS_PATH = "v1/job"
-      TELEMETRY_URL = ENV["TELEMETRY_URL"] # TODO Set to production telemetry URL before release
+      TELEMETRY_URL = if ChefLicensing::Config.license_server_url&.match?("acceptance")
+                        ENV["CHEF_TELEMETRY_URL"]
+                      else
+                        "https://services.chef.io/telemetry/"
+                      end
```

Comment:
Maybe we can refactor this to: `ENV["CHEF_TELEMETRY_URL"] || "https://services.chef.io/telemetry/"` ?

Reply:
No, we do not want the user to override the production URL with any other dummy URL. So this OR condition is not enough.